<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowInfoUI</class>
 <widget class="QDialog" name="ShowInfoUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_userid">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>UserId</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mac">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>0</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>MACAddr</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_userid">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>181</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_macaddr">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_equip_info">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Equipent Information</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_equip_info">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_temperature">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Temperature</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_curr_tem">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Person State</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_person_state">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
